daemon/mgr/container.go

```diff
@@ -589,8 +589,8 @@ func (mgr *ContainerManager) Start(ctx context.Context, id string, options *type
 		return err
 	}
 
-	// check if container's status is running
-	if c.IsRunning() {
+	// check if container's status is running or paused
+	if c.IsRunningOrPaused() {
 		return errors.Wrapf(errtypes.ErrNotModified, "container already started")
```

Comment:
If it is Paused. I do not think the error message should be `container already started`. How about `cannot start a paused container, try unpause instead` like what moby did?

Reply:
Sure